conftest.py

```diff
@@ -227,6 +227,11 @@ def sr_disk(request, host):
             f"disk or block device {disk} is either not present or already used on master host"
     yield disk
 
+@pytest.fixture(scope="session")
+def sr_disk_devid(host, sr_disk):
+    # [5:] remove /dev/ at is not expected by normal sr_disk fixture
+    yield host.disk_devid(sr_disk)[0][len("/dev/"):]
```

Comment:
Are we sure `host.disk_devid` won't return an empty list?
Failures in fixture are really tough to handle.

Reply:
Yes, since `sr_disk` check that the disk exist and any disk that exist will have a symlink in `/dev/disk/by-id/` where we check. `host.disk_devid` returns an array of the `disk/by-id`. From what I currently understand, it should always have at least one. I will be investigating more on that.